### ISSUE TITLE
Develop a proof of concept using CouchDb as a possible sync endpoint for mobile devices

### ISSUE THREAD

**Issue by apeters** [archesproject/arches#2625]
TLDR: http://recordit.co/3dsrpCPqDJ

This proof of concept is really meant to demonstrate the interaction between pouchdb, couchdb, and postgres.  

Make sure you're on this branch to test the demo: https://github.com/archesproject/arches/tree/2625_couch_db_poc

To run this demo, you'll need to first install couchdb - http://couchdb.apache.org/ and install the app, following the instructions.  I set up an admin user on the db using the same credentials as we use for postgres (user: postgres, pwd: postgis).  If you choose to use a different user and/or password, modify the setting COUCHDB_URL setting appropriately.

You'll also have to `pip install couchdb`.

To start, create a new mobile project using the ui.  
![screen shot 2017-11-13 at 1 48 11 pm](https://user-images.githubusercontent.com/823773/32751063-58546630-c879-11e7-895d-576b09312e55.png)
To simulate a resource being added to a project we've hard-coded the core system settings tile as being managed by project you just created.
![screen shot 2017-11-13 at 1 31 21 pm](https://user-images.githubusercontent.com/823773/32750264-ff5c2ea2-c876-11e7-83c3-813c8be91187.png)

The tile data can then be managed and synced here: http://localhost:8000/pouch_demo.

- The very first time the page loads hit the sync button.  
- You should see the system setting information load into the page.  
- Go back to the System Settings page and update one of the values.
- Go back to the demo page and hit the `sync` button again.
- You should observe that the value updates.
- Test that the updates are bi-directional.
- If you click the `listen live` button you can observe that the page is aware of updates that happen via the Fauxton interface (couchdb's built-in web interface) or via the System Settings page.
